apps/jobs/models.py

```diff
@@ -162,3 +162,26 @@ def save(self, *args, **kwargs):
 
         submission_instance = super(Submission, self).save(*args, **kwargs)
         return submission_instance
+
+    @property
+    def submissions_count_on_a_challenge_phase(self):
+        # To get total submissions count on a challenge phase
+        submission = Submission.objects.filter(challenge_phase=self.challenge_phase,
+                                               challenge_phase__challenge=self.challenge_phase.challenge).count()
+        return submission
+
+    @property
+    def submitted_teams_count_on_a_challenge_phase(self):
+        # To get total participant teams count on a challenge phase
+        submission = Submission.objects.filter(challenge_phase=self.challenge_phase,
+                                               challenge_phase__challenge=self.challenge_phase.challenge)
+        submission = submission.values_list('participant_team', flat=True).distinct().count()
+        return submission
+
+    @property
+    def total_submissions_by_a_participant_team_in_a_challenge_phase(self):
```

Comment:
@taranjeet Thanks for pointing this out. @RishabhJain2018 can we use `cached_property` here?